main.ts

```diff
@@ -162,10 +162,13 @@ class Translator {
   visitExternalModuleReferenceExpr(expr: ts.Expression) {
     // TODO: what if this isn't a string literal?
     var moduleName = <ts.StringLiteralExpression>expr;
-    if (!moduleName.text.match(/^\./)) {
-      moduleName.text = 'package:' + moduleName.text;
+    var text = moduleName.text;
+    if (!text.match(/^\./)) {
```

Comment:
Changed to just use substring(2), and inverted the options to have less negation.